jakarta.ls/src/main/resources/transactions.json

```diff
@@ -1,3 +1,14 @@
 {
-
-}
\ No newline at end of file
+    "@Transactional": {
+      "prefix": "@Transactional()",
+      "body": [
+        "@Transactional(",
+        "\trollbackOn={SQLException.class, IOException.class},",
+        "\tdontRollbackOn={IllegalStateException.class})"
+      ],
+      "description": "Specify exception types that will not and will cause a rollback for transaction",
```

Comment:
Would it be better to say "that will and will not"?

Reply:
Fixed!